lib/controllers/migrate-controller.ts

```diff
@@ -1285,6 +1309,48 @@ export class MigrateController
 		return dependencies;
 	}
 
+	private async migrateNativeScriptSvelte(): Promise<IMigrationDependency[]> {
+		const dependencies: IMigrationDependency[] = [
+			{
+				packageName: "svelte-native-nativescript-ui",
```

Comment:
should this instead be `svelte-native` and version `0.9.4` otherwise I can't see where svelte-native is ensured

Reply:
https://github.com/NativeScript/nativescript-cli/pull/5372/files#r506321509